iDiff/cmd/iDiff.go

```diff
@@ -15,7 +17,10 @@ var iDiffCmd = &cobra.Command{
 	Short: "Compare two images.",
 	Long:  `Compares two images using the specifed differ. `,
 	Run: func(cmd *cobra.Command, args []string) {
-		if valid, err := checkArgNum(args); !valid {
+		if validArgNum, err := checkArgNum(args); !validArgNum {
```

Comment:
since all this is doing is checking that the number of args passed in is valid, I think you could combine this with the `checkArgType` method and just call it something like `validateArgs`